Source/Blazorise/Components/Validation/ValidationRule.cs

```diff
@@ -182,5 +184,29 @@ public static void IsFileSelected( ValidatorEventArgs e )
         }
     }
 
+    /// <summary>
+    /// Checks if the date is selected.
+    /// </summary>
+    /// <typeparam name="TValue">Data-type used be binded by the Date property.</typeparam>
+    /// <param name="e"></param>
+    public static void IsDateSelected<TValue>( ValidatorEventArgs e )
+    {
+        var dates = e.Value as TValue[];
+
+        e.Status = dates?.Count( x => !x.IsEqual( default ) ) >= 1 ? ValidationStatus.Success : ValidationStatus.Error;
```

Comment:
`Any?` might shortcut sooner? Also conveys meaning a little bit better?

Reply:
Any might be cleaner, I agree.